.github/workflows/sorn-grab.yml

```diff
@@ -0,0 +1,25 @@
+name: Daily SORN pull
+on:
+  branches: [ main ]
+  schedule:
+    - cron: "11 7 * * *"  # Arbitrary time outside continental US office hours
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to cloud.gov
+    needs: build_and_test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install cf cli
+        uses: ./.github/cf-cli
+      - name: Deploy to cloud.gov
+        env:
+          CF_USERNAME: ${{ secrets.CF_USERNAME }}
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD }}
+          CF_ORG: ${{ secrets.CF_ORG }}
+          CF_SPACE: ${{ secrets.CF_SPACE }}
+          CF_CGHOSTNAME: test
+        run: |
+          ./.cloud-gov/deploy.sh update-sorn
```

Comment:
Can we just have `bundle exec rails cf:on_first_instance federal_register:find_sorns` here instead? Does that work?